projects/ng2-smart-table/src/lib/lib/settings.ts

```diff
@@ -0,0 +1,75 @@
+export interface Settings {
+  columns?: Columns,
+  mode?: 'external' | 'inline';
+  hideHeader?: boolean;
+  hideSubHeader?: boolean;
+  noDataMessage?: string;
+  attr?: Attribute;
+  actions?: Actions;
+  edit?: EditAction;
+  add?: AddAction;
+  delete?: DeleteAction;
+  pager?: Pager;
+  rowClassFunction?: Function
+}
```

Comment:
selectMode?: 'multi';

This 'selectMode' is the configuration for selecting multiple rows in the table using checkbox